torch_geometric/loader/utils.py

```diff
@@ -24,6 +25,22 @@ def index_select(value: Tensor, index: Tensor, dim: int = 0) -> Tensor:
     return torch.index_select(value, 0, index, out=out)
 
 
+def unbatcher(src: torch.tensor, index: torch.tensor, dim: int = -1) -> list:
```

Comment:
Can you add some documentation to this function? I am unsure what's the exact purpose and use-case of this is.

Reply:
After edge classification we ended with data batches, we want to extract edge scores and reconstruct networkx digraphs with this new edge information. This application is very specific for usage in collider physics, where we want to assign two jets to form a parton according to the edge scores.